<?xml version="1.0" encoding="UTF-8" ?>
<launch>

    <!--    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find image_republisher)/config/config.rviz"/>-->

    <node pkg="image_republisher" type="image_republisher" name="image_republisher_node" output="screen">
        <param name="image_dir" value="/home/csl/dataset/LIC-Calib/real_world/calib2" type="string"/>
        <param name="image_topic" value="/camera/image" type="string"/>
        <param name="image_rate" value="10" type="int"/>
    </node>

    <include file="$(find image_republisher)/launch/rosbag-record.launch"/>

</launch>